crash/src/app/data/accidents.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Looks good, we'll leave this one for now till we have the API and Database setup